packages/ts/react-crud/src/autocrud.tsx

```diff
@@ -76,6 +76,24 @@ export type AutoCrudProps<TModel extends AbstractModel = AbstractModel> = Compon
      * property in case a class doesn't have a property matching the defaults.
      */
     itemIdProperty?: string;
+    /**
+     * Whether to show the new button in the toolbar. This is `false` by default,
+     * so that the button is shown.
+     *
+     * NOTE: This only hides the button, it does not prevent from sending a new
+     * item to the service. Make sure that your backend Java service is
+     * properly secured to prevent unauthorized new items.
+     */
+    newButtonHidden?: boolean;
+    /**
+     * Whether to show the toolbar with the new button in the grid. This is
+     * `false` by default, so that the toolbar is shown.
+     *
+     * NOTE: While this also hides the new item button, it does not prevent from
+     * sending a new item to the service. Make sure that your backend Java service
+     * is properly secured to prevent unauthorized new items.
+     */
+    toolbarHidden?: boolean;
```

Comment:
Just a consideration:

We called the property to hide the delete button as `deleteButtonVisible`:
https://github.com/vaadin/hilla/blob/main/packages/ts/react-crud/src/autoform.tsx#L194-L209

And for the things that are enabled by default and the user can disable them, we used the naming as `noHeaderFilters`:
https://github.com/vaadin/hilla/blob/main/packages/ts/react-crud/src/autogrid.tsx#L94-L97

I suggest we keep some sort of harmony for naming the properties, with either of the previously existing ones. I personally am in favor of `noToolbar` and `noNewButton`, as the the presence of the property is enough (`noToolbar` means `noToolbar={true}`),

Reply:
Btw, `noNewButton` is almost the same as `newButtonHidden`, they're both having the default as `false`, and their presence means not showing the UI control they are targeting. The outlier is `deleteButtonVisible` (which should be deprecated in my opinion).

Reply:
Renamed to `newButtonVisible` at the moment, waiting for the discussion about names to finalize.

Reply:
Renamed to `newButton` following internal choice.